hadoopbench/sql/hive_template/bin/hive

```diff
@@ -1,235 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This one is not used?

Reply:
@gczsjdy  yes ,  Native hive can be used, so it is removed.

Reply:
We should put the common scripts in `sql/*/hadoop/run.sh` here